app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -371,6 +372,16 @@ export const CollectionPage = (): React.ReactElement => {
                         <Text variant='small'>Average value</Text>
                       </Stack>
                     </Stack>
+                    { collection.twitterUsername && (
+                      <Stack.Item gutterAfter={PaddingSize.Narrow}>
+                        <Text variant='header3'>Recent Tweets</Text>
+                        <TwitterTimelineEmbed
+                          sourceType='profile'
+                          screenName={`${collection.twitterUsername}`}
+                          options={{ width: 900, height: 500 }}
```

Comment:
can we set the width to 100%? and maybe set a max-width to 900 then. Im sure this is ugly on mobile currently.